doc/index.rst

```diff
@@ -5,10 +5,9 @@
 
 .. container:: index-paragraph
 
-    Nilearn is a Python module for **fast and easy statistical learning on
-    NeuroImaging** data.
+    Nilearn is a Python module to enable **friendly, efficient, and versatile statistical analysis of brain volumes**.
```

Comment:
```suggestion
    Nilearn is a Python module to enable **efficient and approachable statistical learning and analysis of brain volumes**.
```

Reply:
Light suggestion to highlight that it's now both statistical analysis _and_ statistical learning. I also think the documentation should feature in the mission statement (since it's such a big part of Nilearn), so I've added that as well :smile_cat: LMK what you think !

Reply:
I find it clear.